### ISSUE TITLE
Feature Request - customisable extra detail above icons

### ISSUE THREAD

**Issue by sOckhamSter** [ulic75/power-flow-card#68]
Like you've done with battery percentage, allow additional information above all entities. Ideally two lines of text - one being a label, the other being a selectable entity's value such as energy consumed. But, the number of lines should be customisable (one or two are reasonable) which also means the font size should be customisable or dynamic. In my example image, I have shown how this would look displaying the current energy usage for today.
![Power Flow Card Ideas](https://user-images.githubusercontent.com/4546972/169846393-64d12161-b854-44d3-be83-020df4193bbc.jpg)

**Comment by ulic75** [comment]
I think perhaps I'll turn this into a template field and then people can use whatever information they'd like. I'm about to head out on vacation, so I'm not sure when I'll get to this.

**Comment by sOckhamSter** [comment]
I totally agree with that approach - templated values / text would be perfect. If they had unique variable names too so that they could be referenced in a custom CSS component (such as card-mod) then it would be even better so as we could control things like font size too. Have a good holiday :)

**Comment by mariusbaumann** [comment]
I came up with a similar idea, but with a slightly different design approach. 

So the lines/connections get the power values, while the circles contain the kWh values. Additionally there could be numbers for power in the circles too (the combination of the outgoing and incoming connection values, not drawn in my picture). 

There would be some "special effects":

- circle of battery could show the percentage of the charging state. 
- circle of Solar could show how much of the estimated production has already been met. 

![Energy-Diagram](https://user-images.githubusercontent.com/9386539/187316330-3f7836ae-cc63-4da8-9319-3838cc2664fa.jpg)

**Comment by plouf34** [comment]
Did you achieved that?

**Comment by flixlix** [comment]
I created a fork and change a very large portion of the codebase. This new Card includes clickable entities, secondary-info (as requested here) and more:
https://github.com/flixlix/power-flow-card-plus
It is also available in HACS a default repository, so feel free to check it out!
